src/App.jsx

```diff
@@ -14,33 +17,88 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABETICALLY = 'Alphabetically';
+const SORT_FIELD_LENGTH = 'Length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getSortedGoods(goods, sortField, reverse) {
+  let sortedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  switch (sortField) {
+    case SORT_FIELD_ALPHABETICALLY:
+      sortedGoods.sort((a, b) => a.localeCompare(b));
+      break;
```

Comment:
[Non-critical] `sort` method will work the same way if you won't pass any arguments to it